config/test-bluetooth_signal.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "wifi": {
+          "driver": "subt.bluetooth_signal:Bluetooth_signal",
+          "in": [],
+          "out": ["signalStrength"]
```

Comment:
here on the other hand I would use only small letters with underscores ... and again maybe just shorter `signal`